openverse_catalog/dags/common/slack.py

```diff
@@ -252,17 +271,41 @@ def should_send_message(http_conn_id=SLACK_NOTIFICATIONS_CONN_ID):
     except AirflowNotFoundException:
         return False
 
+    # Exit early if this DAG is configured to skip Slack messaging
+    if should_silence_message(text, username, dag_id):
+        log.info(f"Skipping Slack notification for {dag_id}.")
+        return False
+
     # Exit early if we aren't on production or if force alert is not set
     environment = Variable.get("environment", default_var="dev")
     force_message = Variable.get(
-        "slack_message_override", default_var=False, deserialize_json=True
+        "slack_message_override", default_var={}, deserialize_json=True
```

Comment:
Why was this default changed? It seems like the default case would have this return `{}` rather than a bool.

Reply:
I don't know! It looks like this was an accident. Good spot, fixed 😮